database/migrations/2021_07_06_074808_create_tasks_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateTasksTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
```

Comment:
hàm up và down để làm gì e

Reply:
- hàm up : Có tác dụng thực thi migration, đã được khai báo .
- hàm down : Có tác dụng thực thi đoạn lệnh rollback .
-mặc khác, khi chạy php artisan migrate:refresh, nhằm muc đích rollback lại toàn bộ migration của chương trình, và thực hiện câu lệnh migrate. Tức là thực hiện function down() xong rồi thực hiện function up() trong file migration.
Anh cho em gửi